doc/rados/operations/balancer.rst

```diff
@@ -46,14 +46,45 @@ If the cluster is degraded (that is, if an OSD has failed and the system hasn't
 healed itself yet), then the balancer will not make any adjustments to the PG
 distribution.
 
-When the cluster is healthy, the balancer will incrementally move a small
-fraction of unbalanced PGs in order to improve distribution.  This fraction
-will not exceed a certain threshold that defaults to 5%. To adjust this
-``target_max_misplaced_ratio`` threshold setting, run the following command:
+When the cluster is healthy, the balancer will remap
+unbalanced PGs in phases to incrementally improve the uniformity
+of PG distribution.  The maximum percentage of PGs to remap (move) in
+a single phase defaults to 5%. To adjust this
+``target_max_misplaced_ratio`` threshold setting, run a command
+of the following form:
 
    .. prompt:: bash $
 
-      ceph config set mgr target_max_misplaced_ratio .07   # 7%
+      ceph config set mgr target_max_misplaced_ratio .03   # 3%
+
+A larger value may increase the speed of cluster balancing / convergence
+at the potential cost of greater impact on client operations.
+
+There is a separate setting for how uniform the distribution of PGs
+must be for the module to consider adequately balanced.
```

Comment:
The clause "for the module to consider adequately balanced" is not grammatically correct. The form of the clause expects a direct object after the verb "to consider". I suspect that the sentence should be "There is a separate setting for how uniform the distribution of PGs must be in order for the module to consider the cluster adequately balanced", but you'll have to let me know if this is correct.

Anyway, I don't think that this should stop the merging and backporting of this PR, and we can always raise a future PR to make this tiny fix.

Reply:
https://github.com/ceph/ceph/pull/63628